packages/react-components/cypress/utils/storybook.ts

```diff
@@ -0,0 +1,21 @@
+import { manager as knobsManager } from '@storybook/addon-knobs/dist/registerKnobs'
+import type { StoryFile, StoriesWithPartialProps } from '@storybook/testing-react/dist/types'
+
+export function resetKnobs() {
+    knobsManager.knobStore.reset()
+}
+
+export function storiesCompositionToList<TModule extends StoryFile>(
```

Comment:
@pshoniuk this is a really nice way of using `@storybook/testing-react` and Cypress, did you come up with it or is it documented somewhere? 

I see [this](https://github.com/storybookjs/testing-react#batch-testing-all-stories-from-a-file) for batch files in Jest, but don't see much on Cypress in the docs

Reply:
Yes, unfortunately, there is not much information on that. I used the information from the link you shared and from [this article](https://dev.to/mbarzeev/integrating-storybook-with-cypress-and-hmr-4082).

Reply:
@pshoniuk cool, it's a great approach. Only concern is that we're importing from `dist` and not from just `@storybook/testing-react` and wondering if we could ask them to export these types to make it part of the official api

Maybe throw a comment [here](https://github.com/storybookjs/testing-react/issues/47) if you want but up to you

Reply:
I created a feature request in @storybook/testing-react to expose the types we need.